src/style/app.module.css

```diff
@@ -6390,3 +6487,582 @@ button[data-testid='createPostBtn'] {
   font-weight: bold;
   color: var(--black-color);
 }
+
+/* usersidebar  */
+
+.leftDrawer {
+  width: calc(300px);
+  background-color: var(--profile-bg) !important;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: 100;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 1rem 0 1rem;
+  background-color: var(--bs-white);
+  transition: 0.5s;
+  font-family: var(--bs-leftDrawer-font-family);
+}
+
+.leftbarcompheight {
+  display: flex;
+  justify-content: space-between;
+  height: 100vh;
+  padding-bottom: 48px;
+}
+
+@media (max-width: 820px) {
+  .hideElemByDefault {
+    display: none;
+  }
+  .leftDrawer {
+    width: 100%;
+    left: 0;
+    right: 0;
+  }
+  .inactiveDrawer {
+    opacity: 0;
+    left: 0;
+    z-index: -1;
+    animation: closeDrawer 0.4s ease-in-out;
+  }
+  .activeDrawer {
+    display: flex;
+    z-index: 100;
+    animation: openDrawer 0.6s ease-in-out;
+  }
+  .logout {
+    margin-bottom: 2.5rem !important;
+  }
+}
+
+/* ProfileDropdown */
+
+.profileContainer {
+  border: none;
+  padding: 2.1rem 0.5rem;
+  height: 52px;
+  border-radius: 8px 0px 0px 8px;
+  display: flex;
+  align-items: center;
+  background-color: var(--profile-bg) !important;
+  justify-content: space-between;
+  width: 100%;
+  /* Add focus styles for keyboard navigation */
+  &:focus-within {
+    @extend .reusable-focus-visible; /* Referencing the reusable class from the general section */
+  }
+}
+.profilebutton {
+  width: 100%;
+}
+
+.profileContainer:focus {
+  outline: none;
+  background-color: var(--bs-gray-100);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`